uk_election_timetables/elections/city_of_london_local.py

```diff
@@ -21,12 +89,44 @@ def __init__(self, poll_date: date):
     @property
     def sopn_publish_date(self) -> date:
         """
-        Calculate the SOPN publish date for a City of London local election.
+        Calculate the "SOPN publish date" for a City of London local election.
+
+        There are actually 2 releavant dates here:
+        The SOPN is _published_ **17 (working) days before polling day**
+        but then candidates can be withdrawn from it for an additional day
+        after that.
+        So the SOPN becomes _final_ **16 (working) days before polling day**
+
+        For the sake of simplicity, we are going to call 16 days before
+        polling day the "sopn_publish_date".
+
+        As well as the usual exclusions for weekends and bank holidays,
+        the City of London also exclude a Christmas break and Easter break.
+
+        In this section “the Christmas break” means the period beginning with
+        the last week day before Christmas Day and ending with the first week
+        day after Christmas Day which is not a bank holiday; “the Easter break”
+        means the period beginning with the Thursday before and ending with the
+        Tuesday after Easter Day;
 
         :return: a datetime representing the expected publish date
         """
 
-        return working_days_before(self.poll_date, 17, super()._calendar())
+        # Make a new calendar object so we don't mutate
+        # the shared England & Wales calendar
+        calendar = UnitedKingdomBankHolidays().from_country(Country.ENGLAND)
```

Comment:
I think it would actually be better if every `Election` object didn't share a reference to a single calendar object. I reckon it would be better if they each had their own one, but I've decided not to tackle that issue here.